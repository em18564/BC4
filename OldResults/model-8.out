prob: 0.018000000000000002
0.018000000000000002 : 0.011474609375 - 0.011162186675461741
