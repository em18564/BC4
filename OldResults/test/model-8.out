prob: 0.08
